pydantic_evals/pydantic_evals/demo/time_range/test_cases.yaml

```diff
@@ -0,0 +1,35 @@
+# yaml-language-server: $schema=test_cases_schema.json
+rows:
+- name: Event in 3 days
+  inputs:
+    prompt: Schedule a meeting for the same time in 3 days.
+    now: '2023-10-05T14:00:00Z'
+  metadata: {}
+  expected_output:
+    min_timestamp_with_offset: '2023-10-08T13:55:00Z'
+    max_timestamp_with_offset: '2023-10-08T14:05:00Z'
+    explanation: Assuming the user wants a 10-minute window around 14:00 in 3 days.
```

Comment:
Would be nice to use these

Reply:
yeah the examples need to be better, I wouldn't consider this final